atm.js

```diff
@@ -1,21 +1,38 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-}
-
-function checkingWithdrawal(amount){
-
-}
-
-function savingsDeposit(amount){
-
-}
-
-function savingsWithdrawal(amount){
-
-}
-
-// an eventListerner for each button, each one a "click"
\ No newline at end of file
+	
+var atm = {
+
+	$deposit : $("input#deposit") ,
+	$withdraw : $("input#withdraw") ,
+	$userInput : $("input#input") ,
+	$checkingBalance : $("div#checking_balance") , 
+	$balance : $("div.balance") ,
+
+	actions : function() {	
+		var self = this ;
+		self.$deposit.click( function() {
```

Comment:
Try using `.on()` instead of `.click()` - http://stackoverflow.com/a/11878976/850825